test/JDBC/expected/BABEL_OBJECT_ID-vu-verify.out

```diff
@@ -0,0 +1,449 @@
+GO
+
+GRANT EXECUTE ON babel_object_id_proc1 TO babel_object_id_master_user1;
+GO
+
+GRANT EXECUTE ON babel_object_id_func1 TO babel_object_id_master_user1;
+GO
+
+GRANT SELECT ON babel_object_id_v1 TO babel_object_id_master_user1;
+GO
+
+-- tsql      user=babel_object_id_login1 password=12345678
+-- Note: for triggers and constraint it will check permission of table it belongs
+SELECT current_user;
+GO
+~~START~~
+varchar
+babel_object_id_master_user1
+~~END~~
+
+
+SELECT OBJECT_NAME(OBJECT_ID('babel_object_id_t1'))
+GO
+~~START~~
+varchar
+babel_object_id_t1
+~~END~~
+
+
+SELECT OBJECT_NAME(OBJECT_ID('babel_object_id_proc1'))
+GO
+~~START~~
+varchar
+babel_object_id_proc1
+~~END~~
+
+
+SELECT OBJECT_NAME(OBJECT_ID('babel_object_id_func1'))
+GO
+~~START~~
+varchar
+babel_object_id_func1
+~~END~~
+
+
+SELECT OBJECT_NAME(OBJECT_ID('babel_object_id_v1'))
+GO
+~~START~~
+varchar
+babel_object_id_v1
+~~END~~
+
+
+SELECT (CASE WHEN OBJECT_ID('babel_object_id_trg') = NULL THEN 'false' ELSE 'true' END) result;
+GO
+~~START~~
+text
+true
+~~END~~
+
+
+
+-- tsql
+-- Revoke permission from user
+REVOKE SELECT ON babel_object_id_t1 FROM babel_object_id_master_user1;
+GO
+
+REVOKE EXECUTE ON babel_object_id_proc1 FROM babel_object_id_master_user1;
+GO
+
+REVOKE EXECUTE ON babel_object_id_func1 FROM babel_object_id_master_user1;
+GO
+
+REVOKE SELECT ON babel_object_id_v1 FROM babel_object_id_master_user1;
+GO
+
+
+-- tsql  user=babel_object_id_login1 password=12345678
+-- test cross database lookup
+USE master;
+GO
+
+SELECT current_user;
+GO
+~~START~~
+varchar
+babel_object_id_master_user1
+~~END~~
+
+
+-- following should return NULL, as user in db corresponding to current login still don't have permission
+SELECT OBJECT_ID('babel_object_id_db..babel_object_id_db_t1')
+GO
+~~START~~
+int
+<NULL>
+~~END~~
+
+
+-- tsql
+-- grant permission for access
+USE babel_object_id_db
+GO
+
+GRANT SELECT ON babel_object_id_db_t1 TO babel_object_id_user2;
+GO
+
+-- tsql    user=babel_object_id_login1 password=12345678
+USE master;
+GO
+
+SELECT current_user;
+GO
+~~START~~
+varchar
+babel_object_id_master_user1
+~~END~~
+
+
+-- Now we can access
+SELECT OBJECT_NAME(OBJECT_ID('babel_object_id_db..babel_object_id_db_t1'))
+GO
+~~START~~
+varchar
+babel_object_id_db_t1
+~~END~~
+
+
+-- tsql
+USE babel_object_id_db
+GO
+
+REVOKE SELECT ON babel_object_id_db_t1 FROM babel_object_id_user2;
+GO
+
+DROP USER babel_object_id_user2
+GO
+
+USE master;
+GO
+
+DROP USER babel_object_id_master_user1
+GO
+
+-- psql
+
+-- Need to terminate active session before cleaning up the login
+SELECT pg_terminate_backend(pid) FROM pg_stat_get_activity(NULL)
+WHERE sys.suser_name(usesysid) = 'babel_object_id_login1'
+AND backend_type = 'client backend' AND usesysid IS NOT NULL;
+GO
+~~START~~
+bool
+t
+~~END~~
+
+-- Wait to sync with another session
+SELECT pg_sleep(1);
+GO
+~~START~~
+void
+
+~~END~~
+
+
+-- tsql
+DROP LOGIN babel_object_id_login1;
+GO
```

Comment:
Please move it to cleanup file. If you move it there then whole psql code block about terminating session is no longer needed.